test/model/test/model_checkpoint/main.cpp

```diff
@@ -284,10 +284,8 @@ test_checkpoint_wt(void)
     wt_model_ckpt_assert(table, uri, "ckpt3", key2);
     wt_model_ckpt_assert(table, uri, "ckpt3", key3);
 
-    /*
-     * Test assigning the stable timestamp the same value again - this should not have any effect.
-     */
-    wt_model_set_stable_timestamp_both(60);
+    /* Test moving the stable timestamp backwards - this should fail. */
+    wt_model_set_stable_timestamp_both(50);
```

Comment:
Same deal - `testutil_check`?

Reply:
The same for `wt_model_set_stable_timestamp_both`.